config/images/images.yaml

```diff
@@ -164,6 +164,18 @@ images:
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/opentelemetry-operator/opentelemetry-operator
   tags:
   - v0.127.0
+- source: persesdev/perses
+  destination: europe-docker.pkg.dev/gardener-project/releases/3rd/persesdev/perses
+  tags:
+  - v0.50.3
+  - v0.51.0
+- source: persesdev/perses-operator
+  destination: europe-docker.pkg.dev/gardener-project/releases/3rd/persesdev/perses-operator
+  tags:
+  - v0.1.10
```

Comment:
Let's only replicate tags which are used in the master branch and in the release-v1.121 branch.